test/hw7/test_task1_recdict.py

```diff
@@ -1,34 +1,25 @@
+import json
+import os
 from typing import NoReturn, Optional
 
 import pytest
 
 from homework.hw7.task1_recdict import find_occurrences
 
 
-example_tree = {
-    "first": ["RED", "BLUE"],
-    "second": {
-        "simple_key": ["simple", "list", "of", "RED", "valued"],
-    },
-    "third": {
-        "abc": "BLUE",
-        "jhl": "RED",
-        "complex_key": {
-            "key1": "value1",
-            "key2": "RED",
-            "key3": ["a", "lot", "of", "values", {"nested_key": "RED"}],
-        },
-    },
-    "fourth": "RED",
-}
+def example_tree_build():
+    with open(os.path.join(os.getcwd(), "test", "hw7", "tree.json")) as f:
+        return json.load(f)
+
```

Comment:
Is that right way to implement it? and doesn't @pytest.mark.parametrize already making fixtures from what it takes inside it?

Reply:
With fixtures we can get some extra possibilities such as session scopes to preparare object only once